packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/ChangePlanModal.tsx

```diff
@@ -102,13 +102,7 @@ const ChangeProductModal = ({ onClose }: IChangeProductModal) => {
         inner: classes.inner
       }}
       onClose={onClose}
-      testId={
-        (slide === "select")
-          ? "select-plan"
-          : slide === "downgradeDetails"
-            ? "downgrade-details"
-            : "plan-details"
-      }
+      testId={`${ slide }`}
```

Comment:
Please let me know if anyone can think of a better thing to do here. 

I decided to do it like this as I wanted each modal in the flow to have a unique identifier and I had to add more (for select payment, confirm plan and plan success). 

The only other alternative seemed to be to have an even uglier if statement, as I seem to have discovered there's no straightforward way to use a switch in a react component (is that true?).

Reply:
Looks good to me. We can only have ternary statements in react indeed, and having a dedicated function for a test name wouldn't really make more sense I guess. Just a small syntax simplification suggestion, since `slide` is already a string.
```suggestion
      testId={slide}
```

Reply:
@Tbaut Ah yes, thank you 👍